sdk/sig/itly.rbs

```diff
@@ -1,6 +1,4 @@
 class Itly
-  include Itly::Plugins
```

Comment:
@falconandy why remove this?

Reply:
With the line I have a error:
```
$ bin/rspec --format documentation --order defined

An error occurred while loading spec_helper.
Failure/Error: end

RBS::MixinClassError:
  ./sig/itly.rbs:2:2...2:23: Cannot include a class `::Itly::Plugins` in the definition of `::Itly`
```

The `Itly::Plugins` specification is empty:
```
class Itly
  class Plugins
  end
end
```